src/components/asset-list/RecyclerAssetList2/Claimable.tsx

```diff
@@ -62,10 +107,10 @@ export const Claimable = React.memo(function Claimable({ uniqueId, extendedState
             numberOfLines={1}
             style={{ maxWidth: deviceUtils.dimensions.width - 220 }}
           >
-            {claimable.name}
+            {isETHRewards ? 'Rainbow ETH Rewards' : claimable?.name}
           </Text>
           <Text weight="semibold" color="labelTertiary" size="13pt">
-            {claimable.value.claimAsset.display}
+            {isETHRewards ? claimableETHRewardsDisplay : claimable?.value.claimAsset.display}
```

Comment:
what happens if `claimable?.value.claimAsset.display` is undefined here? Maybe will cause a crash